_includes/network-status.md

```diff
@@ -2,13 +2,14 @@
 <a target='_blank' href="https://stats.ffmuc.net/dashboard/db/network-overview">>> Gateway Stats-Dashboard</a>
 <br>
 <br><u><b>Aktuell:</b></u>
+<br>Aktuell sind uns keine Störungen bekannt
 <br>
 <br><u><b>History:</b></u>
-<br><b>01.10.20 10:20-13:46</b> Ausfall von DoT (dot.ffmuc.net)
-<br><b>04.05.20 17:20-17:50</b> IPv4 DHCP gestört für ca. 30 Minuten
-<br><b>03.06 - 06.06.2019</b> Mailserver offline. u.A. Tickethandling, Registrierungen im Chat sind nicht möglich
-<br><b>27.03.2019 17:00-20:00</b> Das gesamte Netz ist aufgrund eines geplanten Serverumzugs offline
+<br><b>01.10.2020 10:20 - 13:46</b> Ausfall vom DoT (dot.ffmuc.net)
```

Comment:
I say `von` is correct but as it's english in a german sentence whatevs.

Reply:
Alright :D